### ISSUE TITLE
Autosave doesn't save current version edited

### ISSUE THREAD

**Issue by D0J0P** [elementary/photos#652]
### What Happened?

When I cropped a photo in Photos, I closed the app and then tried to upload the photo online, and it didn't seem to have the cropped version. It only had the original one. It also doesn't show the cropped version in Files either. If there was a save button inside the app, I could use that, but the app is supposed to autosave. I was able to open the picture in Files' Photo Viewer and press save, but couldn't save it in Photos.

### Steps to Reproduce

- Open Photos
- Edit a picture, then close it.
- Upload it or open it in Files and see that only the original version shows up.

### Expected Behavior

That Photos would autosave the edited version and it would show up in Files and File Roller.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by jeremypw** [comment]
Yes, the original image is kept on disc so you can revert to the original.  The only way I could see immediately to save the modified version was to choose "Other Actions/Export" from the image context menu which is not exactly obvious.  You can then either overwrite the original or choose a different name/location.  I have a feeling that there used to be an easier way than this or the app prompted you if you closed it without saving modifications.

Needs design team input, but one solution would be to have a "Save modified image as" button in the editing page toolbar which appears/becomes sensitive when the image has modifications.
